vultr/terraform/cluster.tf

```diff
@@ -0,0 +1,24 @@
+resource "vultr_kubernetes" "this" {
```

Comment:
Please pick a better name for the resource instead of `this`.

Reply:
re-named